lib/ansible/config/manager.py

```diff
@@ -302,6 +322,42 @@ def __init__(self, conf_file=None, defs_file=None):
         # ensure we always have config def entry
         self._base_defs['CONFIG_FILE'] = {'default': None, 'type': 'path'}
 
+    def load_galaxy_server_defs(self, server_list, default_timeout):
```

Comment:
It seems weird that we would expose the default timeout as an argument here. What was the reasoning behind adding that? Is it because we can't rely on `C.GALAXY_SERVER_TIMEOUT` being set anymore as the underlying dict definition is run from config manager?

Reply:
Mostly i wanted to avoid having the code require a circular dependency on constants, I could have had it call itself, but this also made it easier to override in case it was needed and the callers would always have access to constants at that point.

Reply:
could still make it clearer default:
` default_timeout=self.get_config_value('GALAXY_TIMEOUT')):`